mailbox/store/src/test/java/org/apache/james/mailbox/store/probe/SieveProbeTest.java

```diff
@@ -0,0 +1,23 @@
+package org.apache.james.mailbox.store.probe;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SieveProbeTest {
+
+    @Test
+    public void testSanitizeFileNameWithValidInputForSieveProbe() {
```

Comment:
we use a common naming scheme like "methodShould*When"
In this case, it should be named sanitizeFilenameShouldNotChangeInputWhenSieveExtension

Reply:
Will do.

Reply:
Done.